adapters/Maio/MaioAdapter/GADMediationAdapterMaio.m

```diff
@@ -91,12 +103,33 @@ + (GADVersionNumber)adapterVersion {
   return version;
```

Comment:
🎨 Would you please use `MaioVersion` in adSDKVersion if possible? Because it looks outwardly like sdk v2.